tests/Shipment/ShipmentTest.php

```diff
@@ -235,58 +244,72 @@ public function test_it_should_transform_shipment_to_json_api_with_all_inputs():
                     'last_name'    => $recipientLastName,
                 ],
-                'sender_address'      => [
+                'sender_address'                       => [
                     'street_1'     => $senderStreet1,
                     'city'         => $senderCity,
                     'country_code' => $senderCountryCode,
                     'first_name'   => $senderFirstName,
                     'last_name'    => $senderLastName,
                 ],
-                'return_address'      => [
+                'return_address'                       => [
                     'street_1'     => $returnStreet1,
                     'city'         => $returnCity,
                     'country_code' => $returnCountryCode,
                     'first_name'   => $returnFirstName,
                     'last_name'    => $returnLastName,
                 ],
-                'description'         => $description,
-                'customer_reference'  => $customerReference,
-                'channel'             => $channel,
-                'total_value'         => [
+                'description'                          => $description,
+                'customer_reference'                   => $customerReference,
+                'channel'                              => $channel,
+                'total_value'                          => [
                     'amount'   => $amount,
                     'currency' => $currencyCode,
                 ],
-                'price'               => [
+                'price'                                => [
                     'amount'   => $amount,
                     'currency' => $currencyCode,
                 ],
-                'physical_properties' => [
+                'physical_properties'                  => [
                     'weight' => $weight,
                 ],
-                'customs'             => [
+                'customs'                              => [
                     'content_type'       => 'test',
                     'invoice_number'     => 'test2',
                     'non_delivery'       => 'test3',
                     'incoterm'           => 'test4',
                     'license_number'     => 'test5',
                     'certificate_number' => 'test6',
                 ],
-                'tags'                => [$tag],
+                'sender_tax_identification_numbers'    => [
+                    'country_code' => 'UK',
+                    'Description'  => 'The IOSS number for the Kazan company',
+                    'number'       => 'KSI818298',
+                    'type'         => 'ioss',
+                ],
+                'recipient_tax_identification_numbers' => [
+                    'country_code' => 'UK',
+                    'Description'  => 'The IOSS number for the Kazan company',
+                    'number'       => 'KSI818298',
+                    'type'         => 'ioss',
+                ],
```

Comment:
This seems wrong - it says numbers, while the contents is actually a single tax identification number entry.
```suggestion
                'sender_tax_identification_numbers'    => [
                    [
                        'country_code' => 'UK',
                        'Description'  => 'The IOSS number for the Kazan company',
                        'number'       => 'KSI818298',
                        'type'         => 'ioss',
                    ]
                ],
                'recipient_tax_identification_numbers' => [
                    [
                        'country_code' => 'UK',
                        'Description'  => 'The IOSS number for the Kazan company',
                        'number'       => 'KSI818298',
                        'type'         => 'ioss',
                    ]
                ],
```